apps/users/templates/users/t-shirt.html

```diff
@@ -40,60 +40,57 @@
             <ul class="formfields">
-              <li{% if form.name.errors %} class="error"{% endif %}>
+              <li>
                 <label for="id_name">{{ _('Full Name') }} {{ required() }}</label>
-                {{ form.name }}
-                {{ form.name.errors }}
+                <input name="entry.1631216896" id="id_name" aria-required="true" type="text">
               </li>
-              <li{% if form.address1.errors %} class="error"{% endif %}>
+              <li>
                 <label for="id_address1">{{ _('Address Line 1') }} {{ required() }}</label>
-                {{ form.address1 }}
-                {{ form.address1.errors }}
+                <input name="entry.1645850016" id="id_address1" aria-required="true" type="text">
               </li>
-              <li{% if form.address2.errors %} class="error"{% endif %}>
+              <li>
                 <label for="id_address2">{{ _('Address Line 2') }}</label>
-                {{ form.address2 }}
-                {{ form.address2.errors }}
+                <input name="entry.1195486863" id="id_address2" type="text">
               </li>
-              <li{% if form.city.errors %} class="error"{% endif %}>
+              <li>
                 <label for="id_city">{{ _('City/Town') }} {{ required() }}</label>
-                {{ form.city }}
-                {{ form.city.errors }}
+                <input name="entry.116155144" id="id_city" aria-required="true" type="text">
               </li>
-              <li{% if form.state.errors %} class="error"{% endif %}>
+              <li>
                 <label for="id_state">{{ _('State/Province (if applicable)') }}</label>
-                {{ form.state }}
-                {{ form.state.errors }}
+                <input name="entry.1870171116" id="id_state" type="text">
               </li>
-              <li{% if form.zip.errors %} class="error"{% endif %}>
+              <li>
                 <label for="id_zip">{{ _('Postal Code (if applicable)') }}</label>
-                {{ form.zip }}
-                {{ form.zip.errors }}
+                <input name="entry.51997947" id="id_zip" type="text">
               </li>
-              <li{% if form.country.errors %} class="error"{% endif %}>
+              <li>
                 <label for="id_country">{{ _('Country') }} {{ required() }}</label>
-                {{ form.country }}
-                {{ form.country.errors }}
+                <input name="entry.506045713" id="id_country" aria-required="true" required="" type="text">
               </li>
-              <li{% if form.telephone.errors %} class="error"{% endif %}>
+              <li>
                 <label for="id_telephone">{{ _('Telephone') }} <small>(required by our carrier)</small> {{ required() }}</label>
-                {{ form.telephone }}
-                {{ form.telephone.errors }}
+                <input name="entry.1019342583" id="id_telephone" aria-required="true" required="" type="text">
```

Comment:
It was in the original Google form HTML. I removed it from most, but forgot it here. It doesn't hurt, just unnecessary.